### ISSUE TITLE
without ROI Pool

### ISSUE THREAD

**Issue by 123liluky** [facebookresearch/video-long-term-feature-banks#44]
Is the code convinient for video classification without spatio-temporal action localization?

**Comment by chaoyuaw** [comment]
Yes, this repository supports Charades and EPIC-Kitchens, which are video classification tasks.
